src/makeRobot.js

```diff
@@ -39,6 +39,64 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    version,
+    wheels,
+    'coords': {
```

Comment:
you can remove quotes from `coords`
```suggestion
    coords: {
```